lib/kubernetes-deploy/kubeclient_builder/google_friendly_config.rb

```diff
@@ -13,7 +13,7 @@ def fetch_user_auth_options(user)
       end
 
       def self.read(filename)
-        new(YAML.safe_load(File.read(filename)), File.dirname(filename))
+        new(YAML.safe_load(File.read(filename), [Time]), File.dirname(filename))
```

Comment:
Not sure why I needed this now to run tests, didn't need to when I 🎩 the PR modifying this line. My kubeconfig has changed since then though.